samples/gym-bipedal-walker-sample/README.md

```diff
@@ -0,0 +1,159 @@
+# Bipedal Walker with Stump Track
+
+## Objective
+
+Demonstrate usage of curriculum learning with manual lessons for the challenging task of learning to gait over stump obstacles with randomized height and spacing. Curriculum learning is a Machine Teaching strategy where the agent is gradually introduced to more and more challenging scenarios.
+
+<img src="img/demo_default_stump_gmm_asquad_2.gif" width="400" height="200" />
+
+Often times on problems with highly random simulator configuration variables, learning can struggle to improve. With curriculum learning, the agent in this sample first learns to walk and then the next lesson introduces obstacles. Initially the reward should increase and when the lesson transitions, it is expected for the reward decrease since it is now faced with harder episodes. The reward should increase as training progresses to a higher value than without using the staged learning approach.
+
+[Curriculum Docs](https://docs.microsoft.com/en-us/bonsai/inkling/keywords/curriculum)
+
+### Curriculum Learning
```

Comment:
Might be better to list the config variables earlier so users know which ones are available when defining scenarios?